.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-ci:
+    name: Run
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [5.6, 7.0, 7.1]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          extensions: soap
```

Comment:
It's the version of https://github.com/php-vcr/php-vcr that we use that needs `soap`, btw

Reply:
More on that subject, `soap` is pre-installed for 7.1, but not for 5.6 and 7.0, see https://github.com/shivammathur/setup-php/wiki/Php-extensions-loaded-on-ubuntu-18.04